src/amuse/units/derivedsi.py

```diff
@@ -1,7 +1,7 @@
 from amuse.units.si import *
 
-Hz = named('hertz', 'Hz', 1/s)
-MHz = named('megahertz', 'MHz', 1e6*Hz)
+Hz = named('hertz', 'Hz', (1/s).as_unit())
```

Comment:
maybe put the as_unit in named() function?

Reply:
good idea!

Reply:
done in latest commit